jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/HttpFields.java

```diff
@@ -618,6 +641,8 @@ else if (seenByHeader.add(header))
             }
         }
 
+        Set<String> seenByName = buildByName;
```

Comment:
Nit: you could make `Set<String> seenByName` a final field of the anonymous AbstractSet.

Reply:
@lorban actually I can't, as buildByName is not itself final, it can't be used to initialze even a final field in the inner class.